tests/scripts/set_psa_test_dependencies.py

```diff
@@ -22,6 +22,23 @@
 import re
 import sys
 
+def is_systematic_dependency(dep):
```

Comment:
This function lists whatever dependencies the script can return automatically. The fact that these are exactly the `PSA_WANT_xxx` dependencies is a property of the current implementation rather than the goal of the function.